src/tests/test_dynamic_api/test_serializers.py

```diff
@@ -1049,17 +1053,27 @@ def test_request_unprotected_fields_without_scopes_should_be_allowed(
         complexen_data = complexen_model.objects.create(
             identificatie="AB",
             naam="Paleis",
-            beschrijving="prachtig complex",
+            beschrijving="prachtig complex (geheim)",
+            beschrijving_publiek="prachtig complex",
         )
         monumenten_data.ligt_in_monumenten_complex = complexen_data
         complexen_data.bestaat_uit_monumenten_monumenten.set([monumenten_data])
 
+        # Nasty trick to avoid fetching prefetched data.
+        # When prefetching, the `DynamicSerializer.get_embedded_objects_by_id`
+        # is not touched.
+        monumenten_data._state.fields_cache.clear()
+        complexen_data._state.fields_cache.clear()
+
+        # We also add a camelCased name to the `_fields`, because this
+        # needs an extra snakecasing treatment in the code to work properly.
         drf_request = to_drf_request(
             api_request_with_scopes(
                 [],
                 data={
                     "_expandScope": "ligtInMonumentenComplex",
-                    "_fields": "ligtInMonumentenComplex.naam",
+                    "_fields": "ligtInMonumentenComplex.naam,"
```

Comment:
Comma komt na quotation mark

Reply:
This is intentional, the string should merge with the string at the next line.